common-pipeline/operators/uihints-v3-schema.json

```diff
@@ -599,6 +599,40 @@
           "type": "object",
           "$ref": "#/definitions/resource_definition"
         },
+        "button": {
+          "description": "Properties for button kind and size. Use when control=button.",
+          "type": "object",
+          "properties": {
+            "kind": {
+              "description": "button kind",
+              "type": "string",
+              "enum": [
+                "primary",
+                "secondary",
+                "tertiary",
+                "ghost",
+                "danger",
+                "danger--tertiary",
+                "danger--ghost"
+              ],
+              "default": "tertiary"
+            },
+            "size": {
+              "description": "button size",
+              "enum": [
```

Comment:
I'm not sure we need all these.  What are the sizes for carbon 10?  I think small and sm are duplicates.

Reply:
[Carbon sizes](https://v10-react.carbondesignsystem.com/iframe.html?id=components-button--default&viewMode=docs&args=#button-size) are `field`, `sm`, `lg`, `xl` and `default`. 

But we use `size="small"` for carbon buttons and it works. So I added support for `small`,`medium`,`large` in addition to above options.